static/css/devhub/base.less

```diff
@@ -0,0 +1,113 @@
+@import '../common/normalize.css';
+@import '../impala/fonts/fira/fira.less';
+
+// Colors
+@primary-font: 'Fira Sans', sans-serif;
+@primary-font-color: #fff;
+
+@button-border: #979797;
+@primary-button-background: #29c978;
+@primary-section-background: #0568ba;
+
+// Common styles
+h1 {
+  font-size: 40px;
+  line-height: 1.2;
+  margin-bottom: 0;
+  margin-top: 20px;
+  margin-top: 0;
+}
+
+section {
+  padding: 0 20px 60px;
+
+  > p {
+    font-weight: 300;
+    margin-top: 10px;
+  }
+}
+
+.Button {
+  border-radius: 2px;
+  border: 1px @button-border;
+  box-shadow: 0 2px 3px 0 rgba(0, 0, 0, 0.3);
+  color: @primary-font-color;
+  display: inline-block;
+  padding: 10px 30px;
+  text-align: center;
+  text-decoration: none;
+}
+
+.Button--primary {
+  background-color: @primary-button-background;
+}
+
+// Fonts
+* {
+  font-family: @primary-font;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.DevHub-Header {
+  background-color: @primary-section-background;
+
+  h1,
+  p {
+    color: @primary-font-color;
+  }
+
+  p {
+    font-weight: 300;
+
+    > a {
+      color: @primary-font-color;
+    }
+  }
+}
+
+.DevHub-Navigation {
+  background-color: @primary-section-background;
+  padding: 0 20px 20px;
+
+  ul {
```

Comment:
heh, I'm already not a big fan of ``.Button--primary`` but like ``.button .primary`` more but I see it's advantages and generally, I prefer one convention to personal preference.

I might be biased in that regard but isn't reducing the nesting already part of premature optimization to some degree?